contracts/contracts/OneTimeSBT.sol

```diff
@@ -205,15 +159,51 @@ contract OneTimeSBT is ERC721Enumerable {
         uint256 _tokenId
     ) public view returns (bool) {
         uint256 expirationDate = sbtExpiration[_tokenId];
-        if (expirationDate < block.timestamp) {
-            return true;
+        return expirationDate > block.timestamp;
+    }
+
+    // internal functions
+    function _convertUintArrayToString(uint[] memory input) internal pure returns (string memory) {
+        bytes memory bytesArray = new bytes(input.length);
+        for (uint i = 0; i < input.length; i++) {
+            bytesArray[i] = bytes1(uint8(input[i]));
         }
-        return false;
+        return string(bytesArray);
+    }
```

Comment:
add new util function which is used for older than.
Since older than is two ascii code which represent older than age, read uint array and just be able to convert in string number with string()